ccx_messaging/command_line.py

```diff
@@ -0,0 +1,86 @@
+"""Handlers for CLI commands and some utility functions."""
+
+import argparse
+import importlib.metadata
+import logging
+import os
+import sys
+
+from app_common_python import isClowderEnabled
+from insights_messaging.appbuilder import AppBuilder
+
+from ccx_messaging.utils.clowder import apply_clowder_config
+from ccx_messaging.utils.logging import setup_watchtower
+from ccx_messaging.utils.sentry import init_sentry
+
+
+def parse_args() -> argparse.Namespace:
+    """Parse the command line options and arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("config", nargs="?", help="Application configuration.")
+    parser.add_argument("--version", help="Show version.", action="store_true")
+    return parser.parse_args()
+
+
+def print_version() -> None:
+    """Log version information."""
+    logger = logging.getLogger(__name__)
+    logger.info(
+        "Python interpreter version: %d.%d.%d",
+        sys.version_info.major,
+        sys.version_info.minor,
+        sys.version_info.micro,
+    )
+    logger.info(
+        "%s version: %s",
+        sys.argv[0],
+        importlib.metadata.version("ccx-messaging"),
+    )
+
+
+def apply_config(config) -> int:
```

Comment:
why `int` and not `bool`?

Reply:
Because I want to use the returned value as exit code for the program in any case

Reply:
yes but:

```
>>> True==1
True
>>> False==0
True
```

(but it is just nitpick for years' end :)